addons/web/static/src/core/tree_editor/tree_editor_operator_editor.js

```diff
@@ -1,44 +1,10 @@
 /** @odoo-module **/
 
 import { _t } from "@web/core/l10n/translation";
-import {
-    formatValue,
-    TERM_OPERATORS_NEGATION,
-    toValue,
-} from "@web/core/tree_editor/condition_tree";
-import { sprintf } from "@web/core/utils/strings";
+import { formatValue, toValue } from "@web/core/tree_editor/condition_tree";
 import { parseExpr } from "@web/core/py_js/py";
 import { Select } from "@web/core/tree_editor/tree_editor_components";
```

Comment:
put DomainFacets in another file to avoid cycle in dependencies and do not move these utils